Hardware/index.md

```diff
@@ -38,6 +38,23 @@ The seL4 proofs only hold for specific configurations, as noted in the *Verifica
 
 More information can be found on the [Verified Configurations](../projects/sel4/verified-configurations.md) page.
 
+### Not in the lists below?
+
+If the platform, architecture, feature that you are after is not listed below, your
+options are:
+
+* contact one of the seL4 Foundation [Endorsed Services
+  Providers](https://sel4.systems/Foundation/Services/) to get commercial
+  support to develop such a port or feature;
+* check the [roadmap](../projects/roadmap.html) for any planned contributions,
+  from the seL4 Foundation or larger community, such on any new architecture
+  ports, new large formal verifications, or large or fundamental new features.
+* use one of our [communication channels](https://sel4.systems/contact/) to
+  inquire information, from the seL4 Foundation or the larger community, on
+  implementing such a port or feature;
+* develop the seL4 port or feature yourself, following our [guidelines for
+  contributing kernel code](../projects/sel4/kernel-contribution.html).
```

Comment:
Same as above, we are not an OSS project that will just take new features (esp without verification) -- it needs discussion and an RFC first, and a lot of the time it turns out that existing features are sufficient to solve the problem. The linked page says this (RFC/discussion), but the list here makes it sound like we do want to encourage contributions without talking first.

Platform/BSP ports are different, because they are easy to maintain, but they still need tests and CI, so even there it is not completely unproblematic. New architectures are unlikely to get accepted without major maintenance and expertise commitment. Features are possible, but need discussion and design.

Reply:
@lsf37 thanks I have added explanation in the intro, re-ordered and rephrased a bit the list. Please let me know if it's better